coffee_core/src/coffee.rs

```diff
@@ -90,26 +90,26 @@ impl CoffeeManager {
     /// when coffee is configured, run an inventory to collect all the necessary information
     /// about the coffee ecosystem.
     async fn inventory(&mut self) -> Result<(), CoffeeError> {
-        self.storage
+        let _ = self
+            .storage
             .load::<CoffeeStorageInfo>(&self.config.network)
             .await
-            .and_then(|store| {
+            .map(|store| {
                 self.config = store.config;
-                Ok(())
             });
```

Comment:
```suggestion
            })?;
```

Do you try to simply return the error?

Reply:
Yes that is a cleaner way. thanks!